restapi.go

```diff
@@ -3500,3 +3500,91 @@ func (s *Session) PollExpire(channelID, messageID string) (msg *Message, err err
 	err = unmarshal(body, &msg)
 	return
 }
+
+// ----------------------------------------------------------------------
+// Functions specific to SKUs
+// ----------------------------------------------------------------------
+
+// SKUs returns all SKUs for a given application.
+// appID : The ID of the application.
+func (s *Session) SKUs(appID string) (skus []*SKU, err error) {
+	endpoint := EndpointSKUs(appID)
+
+	body, err := s.RequestWithBucketID("GET", endpoint, nil, endpoint)
+	if err != nil {
+		return
+	}
+
+	err = unmarshal(body, &skus)
+	return
+}
+
+// ----------------------------------------------------------------------
+// Functions specific to entitlements
+// ----------------------------------------------------------------------
+
+// Entitlements returns all antitlements for a given app, active and expired.
+// appID			: The ID of the application.
+// userID			: Optional user ID to look up for.
+// skuIDs			: Optional array of SKU IDs to check for.
+// before			: Optional timestamp to retrieve Entitlements before this time.
+// after			: Optional timestamp to retrieve Entitlements after this time.
+// limit			: Optional maximum number of entitlements to return (1-100, default 100).
+// guildID			: Optional guild ID to look up for.
+// exclude_ended	: Optional whether or not ended entitlements should be omitted.
+func (s *Session) Entitlements(appID, userID string, skuIDs *[]string, beforeID, afterID string, limit int, guildID string, excludeEnded bool, options ...RequestOption) (entitlements []*Entitlement, err error) {
```

Comment:
Optionally, we could pass these parameters via a struct.

Reply:
`appID` can stay, as it is required, everything else, yeah, into the struct.